features/steps/notification_database.py

```diff
@@ -180,7 +184,8 @@ def insert_rows_into_reported_table(context):
             notification_type = int(row["notification type"])
             state = int(row["state"])
             updated_at = row["updated at"]
-            notified_at = row["notified at"]
+            if not notified_at_set:
+                notified_at = row["notified at"]
```

Comment:
we can do
```
if not notified_at:
    notified_at = row["notified at"]
```
and avoid the `notified_at_set` variable (it would be slightly less confusing IMO)

Reply:
That looks good. Changed :)